cmd/client-fs.go

```diff
@@ -670,9 +670,12 @@ func (f *fsClient) listInRoutine(contentCh chan<- *clientContent) {
 					}
 					continue
 				}
+				// Fallback on broken symlinks.
```

Comment:
@Praveenrajmani I think there is a better solution for this, you can os os.Lstat() instead of os.Stat()

Reply:
Done